docs: add info for raspberrypi distro vlc (#1089)
agherzan/meta-raspberrypi: docs/extra-build-config.md

take advantage of your RTC device. You can do that by checking what is
included/configured in the build system based on the inclusion of `rtc` in
`MACHINE_FEATURES`.

## Raspberry Pi Distro, VLC

> To enabled Raspberry PI Distro, VLC the `meta-openembedded/meta-multimedia` layer must be included.

Reviewer: ```suggestion
To enable Raspberry Pi Distro VLC, the `meta-openembedded/meta-multimedia` layer must be included in your `bblayers.conf`.
```

> VLC does not support HW accelerated video decode through MMAL on a 64-bit OS

Reviewer: ```suggestion
VLC does not support HW accelerated video decode through MMAL on a 64-bit OS.
```

take advantage of your RTC device. You can do that by checking what is
included/configured in the build system based on the inclusion of `rtc` in
`MACHINE_FEATURES`.

> ## Raspberry Pi Distro, VLC

Reviewer: ```suggestion
## Raspberry Pi Distro VLC
```

> To enable Raspberry Pi Distro, VLC the `meta-openembedded/meta-multimedia` layer must be

Reviewer: ```suggestion
To enable Raspberry Pi Distro VLC, the `meta-openembedded/meta-multimedia` layer must be
```
